Add Git prerequisite (#470)
boostercloud/booster: docs/README.md

  for Linux
- [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows

##### Install Git

> To initialize this project in your machine you need to install git 

Reviewer: Perhaps something like:

```suggestion
Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.
```

###### Ubuntu

```shell
sudo apt install git-all
```

###### macOS

```shell
brew install git
```

###### Windows

```shell
choco install git
```

> 

Reviewer: Could you add a comment here regarding that the user has the `user.email` and `user.name` properly configured globally?

Author: I make some changes

  for Linux
- [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows

##### Install Git
Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.

###### Ubuntu

```shell
sudo apt install git-all
```

###### macOS

```shell
brew install git
```

###### Windows

```shell
choco install git
```

After installing git in your machine, you need to take a look in the global variables `user.name` and `user.email`, that should be properly configured globally.
you can can check this page to [config your global variables](https://git-scm.com/docs/git-config)

> 

Reviewer: Awesome @jvmazagao , just a couple more suggestions

Can you write here the two steps to configure this?

Also can you please change `global variables` to `Git configuration variables`?

Author: I will add the steps

  for Linux
- [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows

##### Install Git

> Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.

Reviewer 2: What do you think about saying here that you can skip this with the corresponding flag? Suggestion:
```suggestion
Booster will initialize a Git repository when you create a new project (unless you use the `--skipGit` flag), so it is required that you have it already installed in your system.
```

(I also removed a duplicated "it" you had there 😉 )

  for Linux
- [`nvm-windows`](https://github.com/coreybutler/nvm-windows) - Works with native Windows

##### Install Git
Booster will initialize a Git repository when you create a new project, so it is required that you have it already installed it in your system.

###### Ubuntu

```shell
sudo apt install git-all
```

###### macOS

```shell
brew install git
```

###### Windows

```shell
choco install git
```

###### Git configuration variables

> After installing git in your machine, you need to take a look in the global variables `user.name` and `user.email`, that should be properly configured globally.

Reviewer: ```suggestion
After installing git in your machine, make sure that `user.name` and `user.email` are properly configured.
```

> you can can check this page for more informations: [Git configuration variables](https://git-scm.com/docs/git-config) and below you can configure the variables

Reviewer: ```suggestion
Take a look at the [Git configuration page](https://git-scm.com/docs/git-config) for more info.

To configure them, run in your terminal:
```

> ###### Ubuntu
> 
> ```shell
> git config --global user.name "Your Name Here"
> git config --global user.email "your_email@youremail.com"
> ```
> 
> ###### macOS
> 
> ```shell
> git config --global user.name "Your Name Here"
> git config --global user.email "your_email@youremail.com"
> ```
> 
> ###### Windows
> 
> ```shell
> git config --global user.name "Your Name Here"
> git config --global user.email "your_email@youremail.com"

Reviewer: Those are all the same steps, so we can just leave them once:

```suggestion
```shell
git config --global user.name "Your Name Here"
git config --global user.email "your_email@youremail.com"
```
```
